src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/DynamicInsight.cs

```diff
@@ -5,23 +5,45 @@
 namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
 {
     /// <summary>
-    /// Class representing Dynamic Insight
+    /// Class representing Dynamic Insight. This is an insight that has any rendering type inside of it.
+    /// For example, this will allow you to show a graph or table inside an insight
     /// </summary>
-    public class DynamicInsight : InsightBase
+    public class DynamicInsight
```

Comment:
Since Insights class already has Status, Expanded, And Message why not inherit from Insights class?